### DISCUSSION TITLE
rsnapshot or rclone extension?

### DISCUSSION THREAD

**Discussion by tomillr** [cockpit-project/cockpit#18407] [Ideas]
Would it be possible to include an extension that provides a GUI for a sync or backup tool like rsnapshot and rclone? Might be helpful to manage backups and snapshots.

**Comment by martinpitt** [comment]
We won't include it in this "cockpit" core project itself, but we very much encourage writing extensions, as separate projects. They are much easier to develop and maintain. Please see https://cockpit-project.org/blog/creating-plugins-for-the-cockpit-user-interface.html and https://cockpit-project.org/blog/cockpit-starter-kit.html for details.
